test/e2e/volume_provisioning.go

```diff
@@ -107,7 +107,7 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 
 	framework.KubeDescribe("DynamicProvisioner", func() {
 		It("should create and delete persistent volumes [Slow] [Volume]", func() {
-			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke")
```

Comment:
@divyenpatel is the vsphere CI ready, or should the vsphere changes be left out?